.github/workflows/Dependency-Security-Audit.yml

```
@@ -0,0 +1,26 @@
+name: Dependency Security Audit
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # Runs weekly
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)